src/components/video/_index.scss

```diff
@@ -0,0 +1,18 @@
+@use "../../scss";
+
+.c-video {
+	// first take in the default component aspect-ratio
+	// includes a fallback if no aspect ratio is specified
+	// default of 16:9 from engine-theme-sdk
+	--aspect-ratio: var(--c-video-aspect-ratio, 16 / 9);
+
+	// height is used a percentage of the viewport per the requirements
+	// default of 65 from engine-theme-sdk
+	--height: var(--c-video-height, 65);
+
+	@include scss.component-properties("video");
+
+	aspect-ratio: var(--aspect-ratio);
+	// convert to vh via * 1vh
+	height: calc(var(--height) * 1vh);
+}
```

Comment:
We will need to set a `max-width` of `100%` as the full vertical story is adding scrollbars in cases where the width is not wide enough

Reply:
gotcha will do

Reply:
@matthewroach updated -- thanks